src/to_wiki.js

```diff
@@ -118,23 +168,48 @@ const serializer = new WikiTextSerializer({
    },
 
    text(state, node) {
-      let marks = node.marks || []
+      let marks = node.marks.map(m => m.type.name)
+
+      let firstNeedle = function(haystack, needles) {
+         return needles.reduce((min, needle) =>
+          Math.min(min, haystack.indexOf(needle)), haystack.length)
```

Comment:
Yep, because we're searching for things we got from the array we're searching:
https://github.com/iFixit/prosemirror-wikitext/blob/12434cbb83ba1df1edbb9ba2eed231d083196a2e/src/to_wiki.js#L181
and
 https://github.com/iFixit/prosemirror-wikitext/blob/12434cbb83ba1df1edbb9ba2eed231d083196a2e/src/to_wiki.js#L186